infra/terraform/examples/okta_native/main.tf

```diff
@@ -0,0 +1,13 @@
+module "cloudfront_auth_okta_native" {
+  source = "github.com/iress/cloudfront-auth//infra/terraform/modules/okta_native"
```

Comment:
nit: double slash in path

Reply:
This is intentional. The double slash syntax is the way to specify a path inside the repo: https://www.terraform.io/docs/language/modules/sources.html#modules-in-package-sub-directories

Reply:
Cool, I did wonder if it was intentional somehow but couldn't for the life of me figure out what it meant